facilitator/src/bin/facilitator.rs

```diff
@@ -1306,6 +1594,7 @@ fn intake_transport_from_args(matches: &ArgMatches) -> Result<VerifiableAndDecry
         .values_of("packet-decryption-keys")
         .unwrap()
         .map(|k| {
+            debug!("packet decryption private key: {}", k);
```

Comment:
We shouldn't log this, even at debug level, to avoid the risk of accidentally disclosing private keys into cloud log storage.

Reply:
Ah yeah this was testing code